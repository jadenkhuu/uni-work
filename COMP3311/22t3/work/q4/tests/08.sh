psql racing -c "select * from q4('x') order by horse;"
