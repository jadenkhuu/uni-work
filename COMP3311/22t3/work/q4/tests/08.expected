      horse       | average 
------------------+---------
 Flying Fox       |   11062
 Gluteus Maximus  |     600
 Wear The Fox Hat |  115384
(3 rows)
